Changes

```diff
@@ -1,5 +1,9 @@
 LIST OF CHANGES FOR NPG-QC PACKAGE
 
+ - autoqc
+   - added Broad Institute's RNA-SeQC to list of QC checks. Carried out only on
+     RNA library types and mouse or human samples. Tophat-generated transcriptome
+     and GTF/GFF annotation files are required.
```

Comment:
Bioinformatics limitations of the check should be mentioned in the documentation for the check module. Here we just list the changes to the code.